src/clients/GithubClient.ts

```diff
@@ -1,10 +1,16 @@
 import { Octokit } from "octokit";
-import { WorkflowRun, Issue, PullRequest, ActionRuns } from "../types.js";
+import {
+  WorkflowRun,
+  Issue,
+  PullRequest,
+  ActionRuns,
+  JobRun,
+} from "../types.js";
 import { RequestInterface, RequestParameters } from "@octokit/types";
 
 const ORG = "opensearch-project";
 const DEFAULT_REPO = "Opensearch-Dashboards";
-const BACKFILL = process.env.OS_BACKFILL === "true";
+const BACKFILL = new Date().getDate() % 2 === 0;
```

Comment:
I've updated the action to run weekly on Sunday at midnight. Does that address the concern here?

Reply:
Nice! Yeah, that's perfect, I'll remove this